test/eachrow.jl

```diff
@@ -195,4 +195,31 @@ end
     @test res.b == [2, 5]
 end
 
+@testset "#334" begin
```

Comment:
maybe update the docstring of `@eachrow` and `@eachrow!` with information that what this macro does is that it creates a for loop so, in particular, `break` and `continue` may be used?